iothub/device/src/InternalClient.cs

```diff
@@ -1063,7 +1064,7 @@ private Task EnableMethodAsync(CancellationToken cancellationToken)
         }
 
         /// <summary>
-        /// Sets or clears a callback that will be called whenever the client receives a state update
+        /// Sets a callback that will be called whenever the client receives a state update
```

Comment:
I think the previous description was good -> we called out that "set to null" = clear. What do you think?

Reply:
I actually find the previous one misleading. We also have a statement in line 1068 stating how to clear it.